### DISCUSSION TITLE
Showing Error at Line 76 during building the image

### DISCUSSION THREAD

**Discussion by Riturajvats** [apache/airflow#24646] [Q&A] [answered]
https://github.com/apache/airflow/blob/40765307d8c2febbeaddff61551cbfa9d717522e/Dockerfile#L76

![Screenshot (130)](https://user-images.githubusercontent.com/54302152/175671743-cbee4b0f-a895-451c-b8d2-95c40d20dffe.png)

**Comment by potiuk** [comment, marked as answer]
Airflow 2.3.0+ Dockerfile needs buildkit to build https://airflow.apache.org/docs/docker-stack/changelog.html#airflow-2-3
